src/config/reader.rs

```diff
@@ -205,11 +197,10 @@ impl ConfigReader {
             let schema = &file.content;
 
             // Create initial config module
+            let mut parent_dir = PathBuf::from(&file.path); // path to config
+            parent_dir.pop(); // parent dir
```

Comment:
that's strange that we need to repeat that conversion to dir path again and again

Reply:
I think you are referring to parent_dir.pop() being done twice. 1st one is to remove `*.config` from original path, and 2nd time it's done in `LinkType::Config` because loading after loading config via link operator, we need to pop `*.config` once again

Reply:
yeap, and also that repeats in tests. 

Having the same code all over the place feels like degradation and easier to mistake code.